images/installer/Dockerfile.upi.ci

```diff
@@ -0,0 +1,35 @@
+# This Dockerfile is a used by CI to test UPI platforms for OpenShift Installer
+# It builds an image containing binaries like jq, terraform, awscli, oc, etc. to allow bringing up UPI infrastructure.
+# It also contains the `upi` directory that contains various terraform and cloud formation templates that are used to create infrastructure resources.
+
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+RUN hack/build.sh
+
+FROM centos:7
```

Comment:
Shouldn't this be `registry.svc.ci.openshift.org/openshift/origin-v4.0:base` to match `images/installer/Dockerfile.ci`?  Or, if that is missing important packages, can you say what they are in the commit message?

Reply:
testing locally using `registry.svc.ci.openshift.org/openshift/origin-v4.0:base` fails
```console
Step 9/16 : RUN yum install --setopt=tsflags=nodocs -y     epel-release     gettext     openssh-clients &&     yum update -y &&     yum install --setopt=tsflags=nodocs -y     unzip gzip jq awscli util-linux &&     yum clean all && rm -rf /var/cache/yum/*
 ---> Running in ad5c4106cc0b
Loaded plugins: ovl, product-id, search-disabled-repos, subscription-manager
This system is not registered with an entitlement server. You can use subscription-manager to register.
http://base-4-0.ocp.svc/rhel-fast-datapath/repodata/repomd.xml: [Errno 14] curl#6 - "Could not resolve host: base-4-0.ocp.svc; Unknown error"
Trying other mirror.


 One of the configured repositories failed (rhel-fast-datapath),
 and yum doesn't have enough cached data to continue. At this point the only
 safe thing yum can do is fail. There are a few ways to work "fix" this:

     1. Contact the upstream for the repository and get them to fix the problem.

     2. Reconfigure the baseurl/etc. for the repository, to point to a working
        upstream. This is most often useful if you are using a newer
        distribution release than is supported by the repository (and the
        packages for the previous distribution release still work).

     3. Run the command with the repository temporarily disabled
            yum --disablerepo=rhel-fast-datapath ...

     4. Disable the repository permanently, so yum won't use it by default. Yum
        will then just ignore the repository until you permanently enable it
        again or use --enablerepo for temporary usage:

            yum-config-manager --disable rhel-fast-datapath
        or
            subscription-manager repos --disable=rhel-fast-datapath

     5. Configure the failing repository to be skipped, if it is unavailable.
        Note that yum will try to contact the repo. when it runs most commands,
        so will have to try and fail each time (and thus. yum will be be much
        slower). If it is a very temporary problem though, this is often a nice
        compromise:

            yum-config-manager --save --setopt=rhel-fast-datapath.skip_if_unavailable=true

failure: repodata/repomd.xml from rhel-fast-datapath: [Errno 256] No more mirrors to try.
http://base-4-0.ocp.svc/rhel-fast-datapath/repodata/repomd.xml: [Errno 14] curl#6 - "Could not resolve host: base-4-0.ocp.svc; Unknown error"
The command '/bin/sh -c yum install --setopt=tsflags=nodocs -y     epel-release     gettext     openssh-clients &&     yum update -y &&     yum install --setopt=tsflags=nodocs -y     unzip gzip jq awscli util-linux &&     yum clean all && rm -rf /var/cache/yum/*' returned a non-zero code: 1
```
`"Could not resolve host: base-4-0.ocp.svc` error makes me believe if the Dockerfile can only be built in CI cluster.

:crossed_fingers: we'll know when we merge and try to use https://github.com/openshift/release/pull/3206

Reply:
91cd579 to 512f93d